content/de/blog/v2-release/index.md

```diff
@@ -0,0 +1,43 @@
+{
+    "title": "Datenanfragen.de v2: Nagelneue Oberfläche und Beschwerdegenerator",
```

Comment:
"v2" feels more English to me. I'd say, in German you'd more commonly say "2.0".

```suggestion
    "title": "Datenanfragen.de 2.0: Nagelneue Oberfläche und Beschwerdegenerator",
```